test/integration/Config.ts

```diff
@@ -38,8 +38,8 @@ export function getTestFolder(name: string): string {
   return joinFilePath(__dirname, '../tmp', name);
 }
 
-export function removeFolder(folder: string): void {
-  rimraf.sync(folder, { glob: false });
+export async function removeFolder(folder: string): Promise<void> {
+  await remove(folder);
```

Comment:
Or you could have re-exported that function 😄

Reply:
Kept it like this just in case there is another issue only 1 place needs to change. 😄